package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "soap",
-  "version": "0.45.0",
```

Comment:
please revert this.  maintainers will bump.

Reply:
done!